### DISCUSSION TITLE
Exported Apk not working (Deployed to aws) working fine when ran in emulator.

### DISCUSSION THREAD

**Discussion by Apoorva554** [serverpod/serverpod#618] [Q&A] [answered]
**Describe the bug**
Exported apk from mypod_flutter but when installed in android the server is not responding ( it works fine when ran in emulator) . What are steps involved ? 

**To Reproduce**
1. Navigate to my pod flutter
2. run flutter build apk --release
3. when installed the example in not working 

**Expected behavior**
To run when exported apk

**Serverpod versions**
 -  _CLI Version: 1.0.0.
 - _Library version:1.0.0.

**Platform information**
Windows

**Additional context**

**Comment by vlidholt** [comment]
What does the code that creates your Client object look like? You will need to change it to connect to e.g. `https://api.mydeployment.com/`

**Reply by Apoorva554** [reply]
Yes i have changed in main.dart of mypod_flutter (client var)

**Comment by fischerscode** [comment, marked as answer]
Please ensure the internet permission is present in your `AndroidManifest.xml`. I think it is required (not 100% sure).

https://developer.android.com/training/basics/network-ops/connecting

**Reply by Apoorva554** [reply]
It worked i added to mypod_flutter>andriod>src>main_AndroidManifest.xml 

I have another doubt what are steps involved to add SERVERPOD to  existing project ?
